daemon/list.go

```diff
@@ -53,6 +53,13 @@ func (daemon *Daemon) Containers(config *ContainersConfig) ([]*types.Container,
 
 	if i, ok := psFilters["status"]; ok {
 		for _, value := range i {
+			if value != "created" &&
+				value != "restarting" &&
+				value != "running" &&
+				value != "paused" &&
+				value != "exited" {
+				return nil, errors.New("Unrecognised filter value for status")
+			}
```

Comment:
I kinda agree with @thaJeztah a function in ``state.go`` instead of here as if at some point we change the states (adding one, removing one), we will have to remember to change it there..